sonar-ws/src/main/java/org/sonarqube/ws/client/qualitygate/CreateConditionRequest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarqube.ws.client.qualitygate;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+
+public class CreateConditionRequest {
+
+  private final int qualityGateId;
+  private final String metricKey;
+  private final String operator;
+  private final String warning;
+  private final String error;
+  private final Integer period;
+
+  public CreateConditionRequest(Builder builder) {
+    this.qualityGateId = builder.qualityGateId;
+    this.metricKey = builder.metricKey;
+    this.operator = builder.operator;
+    this.warning = builder.warning;
+    this.error = builder.error;
+    this.period = builder.period;
+  }
+
+  public int getQualityGateId() {
```

Comment:
You're right, but I was too lazy for that... Let's say I let this for the next guy who will work on this :)